src/screen/edit_server.rs

```diff
@@ -125,6 +125,13 @@ impl super::Screen for EditServerEntry {
             .position(0.0, -18.0)
             .attach(&mut *server_address.borrow_mut());
 
+        let save_server_error = ui::TextBuilder::new()
```

Comment:
Can you try only creating this if an error actually occurs?

Reply:
I think this is not possible because to create this we need ownership of `ui_container`, but we cant move the ownership into the closure because `add_click_func()` is defined for a closure with only two arguments. I have taken this solution from `src/screen/login.rs` displays a similar error if login fails. There it is done the same way where the text is always constructed but only displayed if login actually fails

Reply:
Okay fair enough, that's probably something for future work then